.github/workflows/publish-deb.yml

```diff
@@ -2,8 +2,8 @@ name: Build and Publish DEB Package
 
 on:
   push:
-    tags:
-      - "**"
+    tags-ignore:
+      - "apisix-runtime/*"
```

Comment:
How about this?
```suggestion
    tags:
      - "apisix/*"
```

Reply:
Don't we need `apisix-base` and `apisix-dashboard`? https://github.com/api7/apisix-build-tools/blob/389e59601d25d232daf180ee25ab83272902fdfb/.github/workflows/publish.yml#L62

Reply:
No need for apisix-base.
The dashboard has not been used for a long time.

I didn't find any tags with `dashboard/*`.

Reply:
OK